### PR TITLE
Update Hungarian translation

### PR DESCRIPTION
**Pull request by meskobalazs**
Only affects `bundle_hu.properties`. @summoner001: feel free to check if everything is in order before merging.

### CODE REVIEW

**Comment by broganeuquoixe** [comment]
I don't know Hungarian however please don't format the triple dots (`...`) with (`…`) because that breaks the text rendering of the tech font. Always test your bundles before sending a PR.

Please look at the `loading...` text in the following videos

Here it's using `...`

https://github.com/Anuken/Mindustry/assets/161990992/10a2c660-5a69-4839-b3b0-04703b6716df


Here it's with `…`

https://github.com/Anuken/Mindustry/assets/161990992/cf3c320d-816f-455f-9212-a62c62f5e610

**Comment by meskobalazs** [comment]
@broganeuquoixe Thanks, I've fixed this.

File: core/assets/bundles/bundle_hu.properties [hunk 1]
```diff
@@ -926,7 +926,7 @@ stat.tiles = Szükséges talaj
 stat.affinities = Módosító körülmények
 stat.opposites = Eltérések
```
Comment (meskobalazs) [inline core/assets/bundles/bundle_hu.properties:927]: „Ellentettek”

File: core/assets/bundles/bundle_hu.properties [hunk 2]
```diff
@@ -1449,399 +1449,399 @@ unit.evoke.name = Evoke
...
 block.micro-processor.name = Mikroprocesszor
-block.logic-processor.name = Logikai Processzor
-block.hyper-processor.name = Hiper Processzor
-block.logic-display.name = Logikai Kijelző
-block.large-logic-display.name = Nagy Logikai Kijelző
-block.memory-cell.name = Memória Cella
-block.memory-bank.name = Memória Bank
+block.logic-processor.name = Logikai processzor
+block.hyper-processor.name = Hiperprocesszor
+block.logic-display.name = Logikaikijelző
```
Comment (meskobalazs) [inline core/assets/bundles/bundle_hu.properties:1841]: „Logikai kijelző”

**Comment by summoner001** [comment]
> Only affects `bundle_hu.properties`. @summoner001: feel free to check if everything is in order before merging.

Hi! Thank you for your help!
I started to review it. It will take a few days.

**Comment by summoner001** [comment]
Line 234: false letter: ħ
Please check commas before "vagy" and "hogy" words.
If you translate the word cyanogen into its Hungarian equivalent (Dician), please translate all occurrences of the word in the whole text
If you change an already translated word Belső Égetésű Erőmű -> Égetéses Erőmű, please rewrite all its occurrences in the whole text
If you change an already translated word Rotary pump which is translated by professionals as Fogaskerék szivattyú please do not use Forgó szivattyú. Please rewrite all its occurrences in the whole text.

In the at first glance, these would be the most important errors.

+1: Please do not follow the english language file in this case and don't use /u26A0 instead of ⚠ but I've already said that, I'll just write it here.

I will keep checking and write if I find any more errors and bugs.

Nice work. Thank you!

**Comment by summoner001** [comment]
If you change an already translated word Világprocesszor (Világ processzor) -> Világfeldolgozó, please rewrite all its occurrences in the whole text

Have you renamed many buildings? Please rewrite them all so that they always translate to the same word.

Line: 2046 - misspelled word: termelőegysésgek
Line: 2454 - non-matching brackets: [ )

**Comment by meskobalazs** [comment]
I have gone through your feedback, some were trivial, so I won't comment on those, as for the others:

- `vagy` and `hogy`: fixed a couple of cases of `vagy`, `hogy` was correct everywhere as far as I can see
- Rotary pump: I went with `fogaskeres szivattyú`, that sound more natural to me, and it is also used "in the wild"
- Világprocesszor: actually I went with `processzor`, I just forgot to change this one back. So it the same again.
- non-matching brackets: [ ): This is not a mistake, this means that the interval is open at the right side (and closed at the left), so the generated value will in the range of 0 to N-1, and it will never be N.

**Comment by meskobalazs** [comment]
Now the PR is ready to be merged.

**Comment by summoner001** [comment]
@Anuken  :

Ready to merge

**Comment by summoner001** [comment]
Thank you for the fast merge.
Happy Easter!
Have a nice day!

**Pull request merged**